.devcontainer/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM registry.fedoraproject.org/fedora:latest
```

Comment:
Consider using a specific version of Fedora to avoid unexpected incompatibility with a major version change.

`FROM registry.fedoraproject.org/fedora:41`

Reply:
Yeah... That's always a good idea...